lib/data/mailboxMetadataApi.ts

```
@@ -22,16 +22,16 @@ export const getMetadataApiByMailbox = async (mailbox: typeof mailboxes.$inferSe
 
-export const getMetadataApiByMailboxSlug = async (mailboxSlug: string) => {
-  const mailbox = await getMailboxBySlug(mailboxSlug);
+export const getMetadataApi = async () => {
+  const mailbox = await getMailbox();
   if (!mailbox) {
     throw new Error("Mailbox not found");
   }
   return { mailbox, metadataApi: await getMetadataApiByMailbox(mailbox) };
 };
 
 export const createMailboxMetadataApi = async (mailboxSlug: string, params: { url: string }): Promise<void> => {
```

ESLint

[error] 'mailboxSlug' is defined but never used. Allowed unused args must match /^_/u.

(@typescript-eslint/no-unused-vars)